controllers/apps/cluster_controller_test.go

```diff
@@ -1997,9 +1996,10 @@ var _ = Describe("Cluster Controller", func() {
 				})
 			})
 
-			It(fmt.Sprintf("[comp: %s] update kubeblocks-tools image", compName), func() {
-				testUpdateKubeBlocksToolsImage(compName, compDefName)
-			})
+			// TODO: enable it later
+			// It(fmt.Sprintf("[comp: %s] update kubeblocks-tools image", compName), func() {
+			//	testUpdateKubeBlocksToolsImage(compName, compDefName)
```

Comment:
i'd suggest to  use `Skip()` instead.  After each run, we will know how many cases are skipped.

Reply:
Reverted